doc/setup.md

```diff
@@ -90,3 +90,13 @@ provide feedback to the user of the runtime error, as it is impossible to pass
 the error to JavaScript when the environment is terminating. In order to bypass
 this behavior such that the Node process will not terminate, define the
```

Comment:
Here too: "The ... can be defined in order to bypass ... "

Reply:
Addressed in https://github.com/nodejs/node-addon-api/pull/1568/commits/23b57cc93946df51f30cbfada61213be14f61421